### DISCUSSION TITLE
thumbnails for movie files

### DISCUSSION THREAD

**Discussion by qrockz** [AnonymouX47/termvisage#11] [Ideas]
Hey,

would be very cool to have the ability to Browse **movie directories** in TUI thumbnail mode and also have the ability to open them in a video player .
I think thumbnails for video files can be provided by **ffmpeg**.

**Comment by AnonymouX47** [comment]
Hmm... thanks so much.

I'll look into it but no promises though, as ffmpeg is quite an heavy dependency.
